x-pack/plugins/observability_solution/inventory/public/components/entities_grid/index.tsx

```diff
@@ -79,18 +79,19 @@ export function EntitiesGrid({
 
   const renderCellValue = useCallback(
     ({ rowIndex, columnId }: EuiDataGridCellValueElementProps) => {
-      const entity = entities[rowIndex];
+      const entity: Entity = entities[rowIndex];
```

Comment:
You don't need to add type here.

```suggestion
      const entity = entities[rowIndex];
```

Reply:
https://github.com/elastic/kibana/pull/197055/commits/9fe837f8f75f00d54ae6b5d9f269a6ff7426f2c5